docs/usage/configuration-options.md

```diff
@@ -1291,29 +1291,29 @@ A similar one could strip leading `v` prefixes:
 }
 ```
 
-## fetchReleaseNotes
+## fetchChangeLogs
```

Comment:
You're changing some headings, which may be used in links.

Did you check if you need to update any links?

Reply:
@HonkingGoose 
I can't find any usage of "#fetchreleasenotes" in the source code